botx/models/buttons.py

```diff
@@ -22,6 +23,9 @@ class ButtonOptions(BotXBaseModel):
     #: text to be shown in toast (show command body if `alert_text` is `None`).
     alert_text: Optional[str] = None
 
+    #: if `client`, should be processed only by client.
```

Comment:
platform, that handle message from markup. If `bot` command will come to bot, else(`client`) will be executed by client.